data/core/src/commonMain/kotlin/com/tunjid/heron/data/repository/PostRepository.kt

```diff
@@ -99,6 +100,14 @@ import sh.christian.ozone.api.Did
 import sh.christian.ozone.api.Nsid
 import sh.christian.ozone.api.model.Blob
 
+sealed class PostsRequest {
```

Comment:
The `PostsRepository` doesn't handle this type, it should be internal to the posts module.